modules/softwaremanagementplans/slides-smp.md

```diff
@@ -337,13 +202,69 @@ The SMP Guide distinguishes software with low, medium, and high management needs
 
 <!-- .slide: data-state="standard" -->
 
+## Mentimeter: how important is it?
+
+==
+
+<!-- .slide: data-state="standard" -->
+
+Consider the following list of software:
+- [NEMO](https://www.nemo-ocean.eu/), a state of the art modelling framework, used for research and forecasting in ocean and climate sciences.
+- [AlphaFold](https://alphafold.com/), an AI system developed by Google DeepMind that predicts a protein’s 3D structure from its amino acid sequence.
+- [Facebook](https://www.facebook.com/), a social media platform connecting people globally.
+- A Jupyter notebook to get the average length of everyone in this classroom.
+- A software package you yourself have been involved in, if any.
+
+Randomly choose 5 requirements and rate on a scale of 1-5 how important those aspects are to include in the SMP.
+It is not necessary to look any of these up, give your best estimation given the stated information.
+
+Note:
+We would usually do this exercise using a mentimeter survey.
+
+===
+
+<!-- .slide: data-state="standard" -->
+
+## Making the call
+
+<div style="width: 55%; float: left">
+Determine the management level, based on:  
+<br></br>
+
+1. **Purpose**: what is the reason or expected end-use for the software?
+2. **Reliability**: what is the effect of software failure and/or non-maintenance?
+3. **Maintenance**: what is the long-term effort needed to maintain the software?
+4. **Lifetime**: How long do you expect (wish) the software to remain in use?
```

Comment:
I made this addition in #133 , not realizing that the other 3 were as listed in the practical guide. I still think it's a good consideration and would like to leave it, but this now does deviate a bit from the guide. What do you think?

Reply:
thanks, I would have never found that. Yes, would be good to document, but also I don't know how I could have found that from the check's error message...